framework/test/unit/modules/interoperability/mainchain/commands/recover_message.spec.ts

```diff
@@ -264,7 +264,7 @@ describe('MessageRecoveryCommand', () => {
 			);
 		});
 
-		it('should return error if idxs = [0]', async () => {
+		it('should return error if idxs[0] <= 1', async () => {
 			transactionParams.idxs = [0];
```

Comment:
Ideally there would be tests for `0` and `1`. But if you find this too much, then maybe change this value to `1`?